templates/zerver/help/import-your-settings.md

```diff
@@ -0,0 +1,34 @@
+# Import your settings
+
+When you create a new account using an email address already associated with a
+Zulip account in another organization, the registration form will offer to import
+the user settings from one of your existing account profiles.
+
+You can only import personal settings (i.e., you will not be able to import
+organization-specific data such as stream settings or custom profile fields).
```

Comment:
Let's make this into a tip; something like this:

!!! tip ""
    Settings that may not apply to all organizations, such as custom profile fields, will not be imported.